lib/geoip_lookup.rb

```diff
@@ -6,7 +6,7 @@ def self.establish_connection!
   def self.lookup(ip)
     @geoip_db.city(ip)
   rescue SocketError => e
-    Rails.logger.warn("GeoipLookup failed lookup on #{ip}: #{e.message}")
+    Rails.logger.warn "GeoipLookup failed lookup on #{ip}", e
```

Comment:
Can we use `Rails.logger.warn` here instead?

Reply:
You mean `Rails.logger.error`?